frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionPipelineDescription.tsx

```diff
@@ -48,7 +49,11 @@ const ModelVersionPipelineDescription: React.FC<ModelVersionPipelineDescriptionP
     >
       <FlexItem data-testid="pipeline-run-link">Run {renderRunLink} in</FlexItem>
       <FlexItem style={{ display: 'flex' }}>
-        <img style={{ height: 24 }} src={typedObjectImage(ProjectObjectType.project)} alt="" />
+        <TypedObjectIcon
+          resourceType={ProjectObjectType.project}
+          style={{ height: 24, width: 24 }}
+          useTypedColor
```

Comment:
We don't want the `useTypedColor` prop here, please remove it. This prop changes the color of the icon, in this case we want a generically colored icon (black in light mode, white in dark mode). TypedObjectIcon will handle this for us so we don't need anything else special.